src/client/components/RetroCertsRoute/index.js

```diff
@@ -15,16 +15,20 @@ function RetroCertsRoute(props) {
     isProduction,
     pageProps,
     requiresAuthentication,
-    ...routeProps } = props;
+    ...routeProps
+  } = props;
 
   let routeChild = <div>Loading...</div>;
   if (isProduction) {
     routeChild = <PageNotFound />;
-  } else if (!requiresAuthentication || userIsAuthenticated(pageProps.userData)) {
+  } else if (
+    !requiresAuthentication ||
+    userIsAuthenticated(pageProps.userData)
+  ) {
     routeChild = <Component {...pageProps} />;
   } else {
     // eslint-disable-next-line no-console
-    console.log('user not authenticated');
+    console.log("user not authenticated");
```

Comment:
Can you delete this and the comment above for me? I accidentally committed it.

Reply:
Sure!